CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Fixes
+
+- Fixed: SentryHttpFailedRequestHandler disposes Content on .net framework ([#3306](https://github.com/getsentry/sentry-dotnet/pull/3306))
```

Comment:
```suggestion
- The `SentryHttpFailedRequestHandler` now disposes content when targeting .NET framework ([#3306](https://github.com/getsentry/sentry-dotnet/pull/3306))
```

Reply:
Updated the changelog... hopefully this is less ambiguous